kernel/src/arch/x86_64/rackscale/mod.rs

```diff
@@ -22,8 +23,20 @@ pub(crate) mod syscalls;
 use crate::cmdline::Transport;
 use crate::error::KError;
 use crate::fs::{cnrfs, NrLock};
+use crate::memory::mcache::MCache;
+use crate::memory::LARGE_PAGE_SIZE;
 use crate::nr;
 use crate::process::Pid;
+use crate::transport::shmem::SHMEM_REGION;
+
+use dcm::node_registration::dcm_register_node;
+
+/// A cache of 2MiB pages, fits on a 2 MiB page.
+///
+/// Used to allocate remote memory (in large chunks)
+pub type FrameCacheMemslice = MCache<2048, 0>;
```

Comment:
I wasn't sure if this is the best place to put this, but I think it makes more sense to have it somewhere in the rackscale module instead of in the memory module